<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimPage</class>
 <widget class="QMainWindow" name="SimPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string extracomment="aaaa"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>740</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="finish">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>730</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Run all instructions</string>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>740</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Memory</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>941</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QMdiArea" name="mdiArea_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>951</width>
       <height>141</height>
      </rect>
     </property>
     <property name="background">
      <brush brushstyle="CrossPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>L2 Cache</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>941</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QMdiArea" name="mdiArea_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>951</width>
       <height>111</height>
      </rect>
     </property>
     <property name="background">
      <brush brushstyle="CrossPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>941</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>0</y>
       <width>231</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QMdiArea" name="mdiArea_7">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <property name="background">
       <brush brushstyle="CrossPattern">
        <color alpha="255">
         <red>214</red>
         <green>214</green>
         <blue>214</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>L1 Cache</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QMdiArea" name="mdiArea_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>231</width>
        <height>191</height>
       </rect>
      </property>
      <property name="background">
       <brush brushstyle="CrossPattern">
        <color alpha="255">
         <red>214</red>
         <green>214</green>
         <blue>214</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>L1 Cache</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>231</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QMdiArea" name="mdiArea_5">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <property name="background">
       <brush brushstyle="CrossPattern">
        <color alpha="255">
         <red>214</red>
         <green>214</green>
         <blue>214</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>L1 Cache</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_11">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>231</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QMdiArea" name="mdiArea_6">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>20</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <property name="background">
       <brush brushstyle="CrossPattern">
        <color alpha="255">
         <red>214</red>
         <green>214</green>
         <blue>214</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>L1 Cache</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>190</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>190</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>941</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>740</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>941</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Core 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Core 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Core 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>0</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Core 4</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
